buildJar.bash

```diff
@@ -16,5 +16,9 @@ cd $FILE
 
 jar cfm TestTaskBook.jar ../$MANIFEST_FILE ./* 
 
-mv TestTaskBook.jar ../
+cd .. 
+
+mkdir Test
+
+mv TestTaskBook.jar Test
```

Comment:
Revert changes from the `buildjar.bash` file.

Reply:
fix